twake/backend/node/src/services/channels/services/tab/service.ts

```diff
@@ -121,6 +151,17 @@ export class Service implements TabService {
   onDeleted(channel: Channel, tab: ChannelTab): void {
     console.log("Tab deleted", tab);
   }
+
+  async userMember(pk: ChannelTabPrimaryKey, context: ChannelExecutionContext): Promise<ChannelMember>{
```

Comment:
If you want to have a function like this, you can, but you have to name it correctly: `userIsChannelMember`, and you can return the result directly like:

```
async userIsChannelMember(user: string, channel: Channel): Promise<boolean>{
  return !!(await this.members.get({ CHANNEL }));
}
```